### ISSUE TITLE
Problem in POST request when no time window

### ISSUE THREAD

**Issue by javiquinte** [EIDA/routing#63]
The following payload via POST request
```
service=station
format=post
alternative=false
Z3 A151A * * * *

```
gives the error
```$ http POST "http://www.orfeus-eu.org/eidaws/routing/1/query" < request-Alparray.txt
HTTP/1.1 400 Bad Request
Error while converting * to datetime```

**Comment by javiquinte** [comment]
Solved in commit ef3c1e1982315d7439664d9d2e20d2ba9e6f8104 ?
To be tested...

**Comment by javiquinte** [comment]
Fixed
